tests/slow/src/test/scala/tests/BloopPantsSuite.scala

```diff
@@ -51,6 +51,40 @@ class BloopPantsSuite extends FastpassSuite {
       .hasProjectsOnRuntimeClasspath(projects("core:core"))
   }
 
+  test("user supplied project name is not shortened") {
+    val workspace = Workspace(s"""$pantsIni
+                                 |/core/BUILD
+                                 |scala_library(
+                                 |  sources=['*.scala'],
+                                 |)
+                                 |/core/Lib.scala
+                                 |package core
+                                 |object Lib {
+                                 |  def greeting = "Hello from lib!"
+                                 |}
+                                 |/src/BUILD
+                                 |scala_library(
+                                 |  sources=['*.scala'],
+                                 |  dependencies=['core:core']
+                                 |)
+                                 |/src/Main.scala
+                                 |package src
+                                 |object Main extends App {
+                                 |  println(core.Lib.greeting)
+                                 |}
+                                 |""".stripMargin)
+
+    val veryLongName =
+      "This.is.a.very.very.long.project.name.that.contains.exactly.one.hundred.characters_12345678901234567"
```

Comment:
Can you please verify that things still work when the provided name contains slashes? This is why the previous logic was put into place: https://github.com/scalameta/fastpass/commit/f920ca59550703890897deab48f59d87583ff635

Reply:
I have changed the logic so that the slashes are always replaced.